bitnami/thanos/README.md

```diff
@@ -1250,7 +1250,7 @@ This helm chart supports using custom Objstore configuration.
 
 You can specify the Objstore configuration using the `objstoreConfig` parameter.
 
-In addition, you can also set an external Secret with the configuration file. This is done by setting the `existingObjstoreSecret` parameter. Note that this will override the previous option. If needed you can also provide a custom Secret Key with `existingObjstoreSecretItems`, please be aware that the Path of your Secret should be `objstore.yml`.
+In addition, you can also set an external Secret with the configuration file. This is done by setting the `existingConfigSecret` parameter. Note that this will override the previous option. If needed you can also provide a custom Secret Key with `existingConfigSecretItems`, please be aware that the Path of your Secret should be `objstore.yml`.
```

Comment:
What about `bucket-cache.yml`, `index-cache.yml`? Cache config files should have those keys